Update to add the required gettext version to work with autoconf 2.70

Upstream-Status: Pending
RP - 2021/1/28

Index: msmtp-1.8.14/configure.ac
===================================================================
--- msmtp-1.8.14.orig/configure.ac
+++ msmtp-1.8.14/configure.ac
@@ -44,6 +44,7 @@ case "${target}" in *-*-mingw*) LIBS="$L
 
 dnl Gettext
 AM_GNU_GETTEXT([external])
+AM_GNU_GETTEXT_VERSION([0.21])
 
 dnl Headers, functions, and types
 AC_CHECK_HEADERS([sysexits.h netdb.h arpa/inet.h sys/select.h sys/socket.h sys/wait.h netinet/in.h])
